tests/View/ViewBladeCompilerTest.php

```diff
@@ -814,4 +814,28 @@ public function testGetTagsProvider()
             ['(((', ')))'],
         ];
     }
+
+    /**
+     * @expectedException \RuntimeException
```

Comment:
Sadly, a call without the outter parenthesis won't be matched
```
@foreach($ary as $e)
// -> compileForeach("$ary as $e")

@foreach $ary => $e
// -> compileForeach("")
```

Because of that regex: https://github.com/laravel/framework/blob/5.3/src/Illuminate/View/Compilers/BladeCompiler.php#L334

I'll see if something better can be done about it as the matching parenthesis test is done twice, in `compileStatements` and `compileForeach`.